testhiml/health/azure/test_himl.py

```diff
@@ -35,6 +35,7 @@
 
 INEXPENSIVE_TESTING_CLUSTER_NAME = "lite-testing-ds2"
 EXPECTED_QUEUED = "This command will be run in AzureML:"
+GITHUB_SHIBBOLETH = "GITHUB_RUN_ID"  # https://docs.github.com/en/actions/reference/environment-variables
```

Comment:
what's a shibboleth? maybe keep the language simpler for the foreigners like me ;-)

Reply:
Will do. It's like a 'tell' in poker, something that gives away something about yourself unwittingly.  It is derived from a scary bible verse: [Judges 12:6](https://www.biblegateway.com/passage/?search=Judges%2012%3A6&version=KJV).  If Google Translate is to be trusted the equivalent German word is erkennungszeichen.